packages/dcos-integration-test/extra/test_exhibitor.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+import pytest
+
+
+class TestExhibitor:
+
+    def test_permissions(self):
+        """
+        ZooKeeper data files are not accessible
+        """
+        # Verify that a parent directory exists
+        assert os.path.isdir('/var/lib/dcos/exhibitor/zookeeper')
```

Comment:
😞 I'm a little sad that this adds to the issue that our integration tests must be run on a cluster's node. However, not having that is not a requirement and so I will approve - but please keep in mind that it would be nice to have tests which run from laptops.